storage/storage.go

```diff
@@ -220,11 +214,20 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	}, nil
 }
 
-// newGRPCClient creates a new Storage client that initializes a gRPC-based
-// client. Calls that have not been implemented in gRPC will panic.
+// NewGRPCClient creates a new Storage client using the gRPC transport and API.
+// Client methods which have not been implemented in gRPC will return an error.
+// In particular, methods for Cloud Pub/Sub notifications are not supported.
```

Comment:
Re: other particularities that are different between transports (for example: iirc, requests to read ACL fail in JSON when UBLA is enabled but not in GRPC) - should those be documented here and/or in relevant comments at this point?

Reply:
I think maybe this can wait for GA; things are still a little in flux at this point. There are some things which aren't supported yet but are coming soon (e.g. MatchGlob for listing objects)